packages/web/src/pages/saved-page/components/desktop/AlbumsTabPage.tsx

```diff
@@ -25,6 +33,16 @@ export const AlbumsTabPage = () => {
     fetchMore,
     collections: albums
   } = useCollectionsData('album')
+  const selectedCategory = useSelector(getSelectedCategory)
```

Comment:
If selected category is not used elsewhere, you could maybe just do it all in the selector